collections/posts.coffee

```diff
@@ -0,0 +1,28 @@
+
+@PromedPosts = new Meteor.Collection "posts"
+
+if Meteor.isServer
+  ReactiveTable.publish "promedPosts", PromedPosts, {}
+
+  Meteor.publish "promedPosts", (limit, range) ->
+    query = {sourceDate: {$exists: true}}
```

Comment:
Updated in the latest commit.
It also gets normalized into addedDate and PublishDate to match closer to the "articles" schema, as well as leave this open for new sources in the future.